### ISSUE TITLE
[BUG] FCU altitude/vs and speed broken

### ISSUE THREAD

**Issue by cobrafast** [flybywiresim/a32nx#810]
**Mod Version**

Master 31bfec4e90ed79e8afd14b3f5e4990a3ddf09473

**Describe the bug**

* The speed knob can be used to dial a speed, but pulling or pushing it does nothing.
* The altitude knob can be used to dis/engage managed mode, but rotating it does nothing. It's stuck at 5000 ft.
* The altitude knob cannot be switched between 100/1000 ft. No mouse interaction is even present.
* The FMA seems to be stuck in V/S mode
  * Randomly switches to ALT(*) when descending (!), then will hold that altitude, which will likely differ from the selected one, forever since it can't be changed

**To Reproduce**

1. Create a flight plan through the MSFS main menu
2. Start at a parking position and walk through the startup procedure (bats on, ext pwr on, apu start)
3. Enable FD

**Expected behavior**

* A speed different from the planned one can be selected
* An altitude other than 5000 ft can be selected
* The altitude selector can be switched between 100/1000 ft changes
* ALT, CLB, DES modes can be armed and engaged

**Actual behavior**

see above

**References**

—

**Additional context**

—

Discord username (if different from GitHub): Cobra#5370

**Comment by NathanInnes** [comment]
Unable to replicate

**Comment by cobrafast** [comment]
Tried current master (cc1cde12d9cffa1c005040c6264420624df8fc24), problem persists.

Things I've tried to troubleshoot:
* Remove any other community content (which were just sceneries and liveries)
* Move Packages folder to a different SSD
* Move Packages folder to the default location
* Start with or without an IFR flight plan
* Start at a runway instead of a parking position (which turned out to be even more broken, see below)
* Redownload A32NX folder from GitHub-generated zip (instead of copying from a checkout)

Additionally, I noticed that the FMA is stuck in V/S NAV 1FD2 when starting at a runway. Disabling the FD doesn't do anything but clear the green cross from the PFD. The heading knob doesn't make it forget NAV either.

Anything else I can try to remedy this?

**Comment by cobrafast** [comment]
I think I see what the problem is, I still have MSFS version 1.7.14.0 when apparently 1.8.3.0 has been out for a while. MS Store isn't finding any available updates.
I'll try to get that sorted and report back.

**Comment by CaptCQ** [comment]
I have the same issue. I've updated my sim. 

My altitude knob wont change the altitude from 5000ft. Vs doesnt work/respond.

**Comment by MylesKingsnorth** [comment]
I also seem to have this issue, additionally, I can't change the altitude select from 1000s to 100s. I made a quick video demonstrating [https://youtu.be/MYJQYPhdczU](https://youtu.be/MYJQYPhdczU) on master cc1cde12d9cffa1c005040c6264420624df8fc24

**Comment by cobrafast** [comment]
Problems do no longer occur for me after properly updating the sim to version 1.8.3.0.

I had to manually uninstall and (re)install via the Xbox app. Only then did it update the content packages.

**Comment by CaptCQ** [comment]
Mine is steam version. Cant roll back update.

**Comment by cobrafast** [comment]
@CaptCQ 1.8.3.0 should be the very latest version. You could try the usual Steam things for stuck updates, like restarting Steam or doing the verify integrity of game files thing.

**Comment by bam1220** [comment]
I had the same issue with not being able to set the altitude or the VS. I was also having issues where my checklist and AI for the drop down menu were greyed out. I ended up uninstalling the sim. I then reinstalled it and all is well. I can now set the altitude instead of it being stuck at 5000 ft. VS also works as does all my drop down menus. The reinstall even kept all of my controller settings. I am using the Xbox for PC version. I signed up here to post this in case it helps anyone that has this issue. I hope it helps.
